framework/doc/content/documentation/systems/MeshModifiers/BreakMeshByBlock.md

```diff
@@ -0,0 +1,36 @@
+# Break Mesh By Block
+
+!syntax description /MeshModifiers/BreakMeshByBlock
+
+This class implement a methods to split a monolithic mesh by blocks and follows the
+method proposed by VP Nguyen [cite:Nguyen2014]
+
+To split the mesh nodes belonging to a boundary between 2 blocks are duplicated, the mesh is split and a new sideset is added. The new sideset is the boundary where the Interface lives.
+
+
+
+As an option, the interface can be split into $N$ different sidesets. $N$ is the number of adjacent block pairs. This is achieved by setting  `split_interface=true`. This is useful when modeling interfaces with different parameters.
+
+## Example Input File Syntax
+
+
+
+### Single interface
+
+!listing test/tests/mesh_modifiers/break_mesh_by_blocks/2D/TestBreakMesh_2DJunction_Auto.i block=MeshModifiers
+
+### Multiple interfaces
+
+When `split_interface=true` the new generated interface is splitted by block pairs:
+
+!listing test/tests/mesh_modifiers/break_mesh_by_blocks/2D/TestBreakMesh_2DJunction_splitTrue_Auto.i block=MeshModifiers
+
+The naming convention for the interface is `bMi_bSj`, where $i$ and $j$ represents the original blocks ids.
```

Comment:
Can you provide an example so that this can be clearer?

Reply:
For instance if block 1 is  neighboring blocks 2 and 3 you will have two new interfaces:
bM1_bS2 and bM1_bS3 both sidesets will be added to block one being them the one with the lower id. 
I will add this to the documentaion